apps/payments/next/middleware.ts

```diff
@@ -7,11 +7,11 @@ export function middleware(request: NextRequest) {
   const nonce = Buffer.from(crypto.randomUUID()).toString('base64');
   const cspHeader = `
     default-src 'self';
-    connect-src 'self' 'https://api.stripe.com';
-    frame-src 'https://js.stripe.com' 'https://hooks.stripe.com';
+    connect-src 'self' https://api.stripe.com;
+    frame-src https://js.stripe.com https://hooks.stripe.com;
     script-src 'self' 'nonce-${nonce}' 'strict-dynamic' https: http: 'unsafe-inline' ${
     process.env.NODE_ENV === 'production' ? '' : `'unsafe-eval'`
```

Comment:
nit: should this be production AND stage?

Reply:
`stage` isn't one of the [standard NODE_ENV values recommended by Next.js](https://nextjs.org/docs/messages/non-standard-node-env), so we should be fine with just `production`.